web3/manager.py

```diff
@@ -60,14 +75,14 @@ def default_middlewares(web3):
         Leaving ens unspecified will prevent the middleware from resolving names.
         '''
         return [
-            (request_parameter_normalizer, 'request_param_normalizer'),
-            (gas_price_strategy_middleware, 'gas_price_strategy'),
-            (name_to_address_middleware(web3), 'name_to_address'),
-            (attrdict_middleware, 'attrdict'),
-            (pythonic_middleware, 'pythonic'),
-            (normalize_errors_middleware, 'normalize_errors'),
-            (validation_middleware, 'validation'),
-            (abi_middleware, 'abi'),
+            (request_parameter_normalizer, 'request_param_normalizer'),  # Delete
+            (gas_price_strategy_middleware, 'gas_price_strategy'),  # Add Async
+            (name_to_address_middleware(web3), 'name_to_address'),  # Add Async
+            (attrdict_middleware, 'attrdict'),  # Delete
+            (pythonic_middleware, 'pythonic'),  # Delete
+            (normalize_errors_middleware, 'normalize_errors'),  # Add async
+            (validation_middleware, 'validation'),  # Add async
+            (abi_middleware, 'abi'),  # Delete
         ]
```

Comment:
These comments are notes to myself.  Will remove.

Reply:
@dylanjw I am planning on taking this over shortly if you don't have objections! When I was looking this over, it looks like the attrdict_middleware didn't get moved over. I assume that the plan is to shift it still? Or did you run into something where it didn't make sense any more?

Reply:
The plan was to move the attrdict_middleware functionality to the method formatters later.  I didnt run into any complications.